### PR TITLE
Ben instructor Instructor Text Box Updates

### PR DESCRIPTION
**Pull request by BenCrutt**
This PR, adds functionality to the Course instructor text box on the course instructor search page.  It also makes it not case sensitive converting all uppercase letters to lowercase ones. Now, the form input will be ready for the backend to build the table. Resolves Issue #37 

Note the submit feature is not ready yet so upon pressing, it currently refreshes the page to lead to a better user experience. Getting the table set up is in the works and once ready the submit button will no longer refresh the page.

### CODE REVIEW

**Review by b4k3s — DISMISSED** [review]
I reviewed your code and tested the functionality of qa deployment. Everything seems to work as you intended for this PR. LGTM.

File: frontend/src/main/components/BasicCourseSearch/InstructorCourseSearchForm.js [hunk 1]
```diff
@@ -20,7 +20,8 @@ const InstructorCourseSearchForm = ({ fetchJSON }) => {
   // Stryker disable all : not sure how to test/mock local storage
```
Comment (pconrad) [inline frontend/src/main/components/BasicCourseSearch/InstructorCourseSearchForm.js:20]: These should be a `// Stryker restore all` somewhere ... maybe line 24?

**Review by pconrad — CHANGES_REQUESTED** [review]
There is a `// Stryker disable all` in front of the lines that use local storage, which is legit because we only recently learned how to mock local storage.

However, there isn't a matching `// Stryker restore all`, which means we are probably missing a lot of mutations that *should* have tests covering them.

When you put this in, you may get some mutation test failures, but those can be dealt with.

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**